UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -33,7 +33,7 @@ public class LengthTests : LengthTestsBase
 
         protected override double FeetInOneMeter => 3.28084;
 
-        protected override double USSurveyFeetInOneMeter => 3.2808333333465;
+        protected override double USSurveyFeetInOneMeter => 3.2808333333;
```

Comment:
Probably a bad commit comment but one of the tests failed because I used a truncated value to get the above value.  Your idea of putting the fractions in is better.   So, it should be 3937/1200